apps/frontend/src/components/home-buttons/twitter-button/twitter-button.css

```diff
@@ -0,0 +1,3 @@
+.twitter-x {
```

Comment:
Maybe it will be better to implement it within the svg with Tailwind?

Reply:
Styled within the svg class with tailwind as advised.